Orm/Xtensive.Orm/Orm/Model/ColumnInfoRef.cs

```diff
@@ -18,52 +18,40 @@ namespace Xtensive.Orm.Model
   /// </summary>
   [Serializable]
   [DebuggerDisplay("TypeName = {TypeName}, FieldName = {FieldName}, ColumnName = {ColumnName}, CultureInfo = {CultureInfo}")]
-  public sealed class ColumnInfoRef: IEquatable<ColumnInfoRef>
+  public readonly struct ColumnInfoRef: IEquatable<ColumnInfoRef>
   {
     /// <summary>
     /// Gets type name of reflecting <see cref="TypeInfo"/>.
     /// </summary>
-    public string TypeName { get; private set; }
+    public string TypeName { get; }
 
     /// <summary>
     /// Gets name of the <see cref="FieldInfo"/>.
     /// </summary>
-    public string FieldName { get; private set; }
+    public string FieldName { get; }
 
     /// <summary>
     /// Gets name of the <see cref="ColumnInfo"/>.
     /// </summary>
-    public string ColumnName { get; private set; }
+    public string ColumnName { get; }
 
     /// <summary>
     /// Gets <see cref="CultureInfo"/> info of the <see cref="ColumnInfo"/>.
     /// </summary>
-    public CultureInfo CultureInfo { get; private set; }
+    public CultureInfo CultureInfo { get; }
+
+    public bool HasValue => TypeName != null;
```

Comment:
There is no XML comment here.
And as shown in other comments we most likely don't need this property.

Reply:
removed it